packages/ui-drilldown/src/Drilldown/index.tsx

```diff
@@ -1418,6 +1422,9 @@ class Drilldown extends Component<DrilldownProps, DrilldownState> {
             })}
           >
             <View
+              themeOverride={{
```

Comment:
we shouldn't use `themeOverride` for our own components

Reply:
I think according to our last discussion with Topi, we haven't made a conclusion that themeOverride should be used or not

Reply:
removed